packages/ckeditor5-utils/src/translation-service.ts

```diff
@@ -177,17 +180,18 @@ export function _translate( language: string, message: Message, quantity: number
 		throw new CKEditorError( 'translation-service-quantity-not-a-number', null, { quantity } );
 	}
 
-	const numberOfLanguages = getNumberOfLanguages();
+	const normalized: Translations = translations || global.window.CKEDITOR_TRANSLATIONS;
```

Comment:
Let's make this variable name to be more descriptive, maybe `normalizedTranslations` would be better?